migrations/plain/src/main/scala/PlainMigrationManager.scala

```diff
@@ -0,0 +1,33 @@
+package scala.migrations.plain
+
+import java.io._
+import scala.io.Source
+import scala.migrations.Migration
+import scala.migrations.MigrationManager
+
+trait PlainMigrationManager extends MigrationManager[Int] {
+  def dblocation = System.getProperty("user.dir")+"/migrationInfo.txt"
```

Comment:
Instead of configuring using a system property, I imagine a user creating a MigrationManager subclass in their own project. If we have an sbt plugin on top that handles the commands, we may need a way to point it to the class in user code that should be used. Maybe this requires a multi-project build, where the user has one migration manager project, potentially a codegen project, certainly another project for their main code.

Offering a pre-configured PlainMigrationManager that is opinionated may be a nice feature though. No need to change, just sharing my vision.